diff -ur a/CMakeLists.txt b/CMakeLists.txt
--- a/CMakeLists.txt	2020-05-05 01:05:13.000000000 +0300
+++ b/CMakeLists.txt	2020-05-22 10:51:41.948867988 +0300
@@ -94,15 +94,6 @@
 
 
 
-# rebuild man pages
-set(MANFILE_FULL "${CMAKE_SOURCE_DIR}/data/cherrytree.1")
-set(MANFILE_FULL_GZ "${MANFILE_FULL}.gz")
-add_custom_command(
-  OUTPUT ${MANFILE_FULL_GZ}
-  COMMAND gzip -f -k --best -n "${MANFILE_FULL}"
-)
-add_custom_target(man_pages ALL DEPENDS ${MANFILE_FULL_GZ})
-
 # install commands
 install(TARGETS cherrytree RUNTIME DESTINATION bin)
 if(${CMAKE_SYSTEM_NAME} MATCHES Linux|.*BSD|DragonFly)
@@ -111,7 +102,7 @@
           DESTINATION "${SHARE_INSTALL}/applications")
   install(FILES "${CMAKE_SOURCE_DIR}/icons/cherrytree.svg"
           DESTINATION "${SHARE_INSTALL}/icons/hicolor/scalable/apps")
-  install(FILES "${MANFILE_FULL_GZ}"
+  install(FILES "${CMAKE_SOURCE_DIR}/data/cherrytree.1"
           DESTINATION "${SHARE_INSTALL}/man/man1")
   install(FILES 
           "${CMAKE_SOURCE_DIR}/data/cherrytree.mime"
